easybuild/easyblocks/c/clang.py

```diff
@@ -87,6 +92,8 @@ def extra_options():
             'skip_sanitizer_tests': [True, "Do not run the sanitizer tests", CUSTOM],
             'default_cuda_capability': [None, "Default CUDA capability specified for clang, e.g. '7.5'", CUSTOM],
             'build_extra_clang_tools': [False, "Build extra Clang tools", CUSTOM],
+            'amd_gfx_list': [None, "List of AMDGPU targets to build for. Possible values: " +
```

Comment:
Do you mind reordering the keys in alphabetical order here while you're adding another custom easyconfig parameter?

Reply:
:+1: